spec/std/http/spec_helper.cr

```diff
@@ -2,7 +2,12 @@ require "spec"
 require "../spec_helper"
 require "../../support/fibers"
 
-private def wait_for(timeout = 5.seconds)
```

Comment:
There was no caller with an explicit timeout so for clarify I moved the argument to inside the method.